### ISSUE TITLE
Buy/sell stocks with tacos

### ISSUE THREAD

**Issue by Armintron** [cs-discord-at-ucf/lion#754]
Utilzing the work done to [get current stock prices with lion](https://github.com/cs-discord-at-ucf/lion/blob/master/src/app/plugins/price.plugin.ts), let users buy/sell stocks using their tacos. Can do some conversion like $1 = 10 tacos or something

**Comment by c650** [comment]
yeah, this seems like a neat little thing. and since it's real money, we dont really have to get it 100% correct (corporate actions, dividends, delists, etc)
